### PR TITLE
docs: Make managementCluster defintion obvious

### PR DESCRIPTION
**Pull request by raykrueger**
managementCluster is a critical field for workload clusters. It was only defined as the last sentence in a note here. Let's make it more obvious.

*Issue #, if available:*
None
*Description of changes:*
managementCluster is a critical field for workload clusters. It was only defined as the last sentence in a note here. Let's make it more obvious.

*Testing (if applicable):*
Markdown preview

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jonathanmeier5 — APPROVED** [review]
/lgtm

**Comment by raykrueger** [comment]
Just noticed I mixxed up mgmt-demo and demo-mgmt in my example... and copied and pasted it across the board.

File: docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md [hunk 1]
```diff
@@ -123,7 +123,21 @@ Follow these steps if you want to use your initial cluster to create and manage
    ```
 
    Refer to the initial config described earlier for the required and optional settings.
-   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Be sure to set the `managementCluster` field to identify the name of the management cluster. Keep the tinkerbellIP of workload cluster the same as tinkerbellIP of the management cluster.
+   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Keep the tinkerbellIP of workload cluster the same as tinkerbellIP of the management cluster.
+
+1. Be sure to set the `managementCluster` field to identify the name of the management cluster.
+
+   For example, the management cluster, _mgmt-demo_ is defined for our workload cluster _demo-workload-cluster_ as follows:
```
Comment (jiayiwang7) [inline docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md:130]: nit: mind if we name mgmt and workload clusters to `_mgmt_` and `_w01_` just to stay consistent with the rest of the doc. Same for other providers

Comment (raykrueger) [reply docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md:130]: Understood. Working on it

Comment (raykrueger) [reply docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md]: Done. I derped up the commits a bit, but should be good now.

**Comment by raykrueger** [comment]
It looks like this may have been merged but never deployed? https://anywhere.eks.amazonaws.com/docs/getting-started/production-environment/vsphere-getstarted/#create-separate-workload-clusters

**Pull request merged**